src/test/java/com/mercadolivre/bootcamp/projeto_integrador/unit/SalesmanServiceTest.java

```diff
@@ -0,0 +1,143 @@
+package com.mercadolivre.bootcamp.projeto_integrador.unit;
+
+import com.mercadolivre.bootcamp.projeto_integrador.entity.Salesman;
+import com.mercadolivre.bootcamp.projeto_integrador.exception.SalesmanDoesNotExistException;
+import com.mercadolivre.bootcamp.projeto_integrador.exception.SalesmanListIsEmptyException;
+import com.mercadolivre.bootcamp.projeto_integrador.repository.SalesmanRepository;
+import com.mercadolivre.bootcamp.projeto_integrador.service.SalesmanServiceImpl;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+
+@ExtendWith(MockitoExtension.class)
+public class SalesmanServiceTest {
+
+    @Mock
+    private SalesmanRepository repository;
+
+    @InjectMocks
+    private SalesmanServiceImpl service;
+
+    @Test
+    @DisplayName("This test should create a salesman.")
```

Comment:
Trocar o começo de toda as frases no DisplayName. Tirar o "this test" e deixar "it should...".